### ISSUE TITLE
Too many "music"

### ISSUE THREAD

**Issue by Apollon-Theos** [mRemoteNG/mRemoteNG#2283]
Every confirmation show dialog and play sound

## Expected Behavior
Sound must by only for "Alarm"

## Current Behavior
Every dialog with confirmation play sound. For work environment this is bad practic.

## Possible Solution
Set option for function "ShowConfirmationDialog" with "NoSound"

## Steps to Reproduce (for bugs)
1. set any options, that needed restart program
2. listen to computer :)
3.
4.

## Context
in work environment must be silence, not many time sounds by every confirmation

## Your Environment

* Version used: 1.77.3
* Operating System and version (e.g. Windows 10 1709 x64):
Windows 10 Ent, Single language, Russian

**Comment by Kvarkas** [comment]
I don't think we can control such, that comes from windows side, but will have a look.

**Comment by Apollon-Theos** [comment]
https://docs.microsoft.com/en-us/office/vba/language/reference/user-interface-help/msgbox-function
similar function with many optional parameters, that control view and possibilities of "Message to User".
I think same way exist in C# (C++)

**Comment by simonai1254** [comment]
Reducing the sound created by an app can be a good idea. Though I don't really care as I just set the windows sound scheme to "No Sound"/mute the system sound channel, which mutes almost all annoying popup-alerts. Upon that I put the speakers on mute (I generally use headphones though), so I don't really care to be honest.

**Comment by Kvarkas** [comment]
Perhaps we can add new section in settings SilentMode = On/Off :)

**Comment by Hakun1n** [comment]
The "Bell" sound after every use of MultiSSH is annoying. I have the "PuTTy bell" already disabled but that does not affect he MultiSSH bar. mRemoteNG is not in Win10 volume mixer so it can't be explicitly muted. As i can see the bell sound belongs to "System sound" (or at least that bar gets bumped up in volume mixes for a second when multiSSH command is sent)

The global "Mute" option would be more than enough to suppress these annoyances :)

Thanks !

**Comment by robertpopa22** [comment]
**Fix available for testing**

A fix for this issue has been implemented in our community fork (`robertpopa22/mRemoteNG`, branch `main`). A beta build including this fix is available from the fork's Releases page.

Please test and report if this resolves your issue.

---

> *We sincerely appreciate the mRemoteNG maintainers and the wonderful community that has kept this project alive and thriving for over a decade. Your work powers thousands of IT professionals worldwide, including us. Our fork builds entirely on your foundation — we're preparing a consolidated Pull Request to contribute these improvements back to the official project. Thank you for everything you've built.*
>
> *— robertpopa22, community contributor*
